code/_globalvars/traits.dm

```diff
@@ -81,7 +81,6 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_NOSELFIGNITION_HEAD_ONLY" = TRAIT_NOSELFIGNITION_HEAD_ONLY,
 		"TRAIT_CONTORTED_BODY" = TRAIT_CONTORTED_BODY,
 		"TRAIT_DEFLECTS_PROJECTILES" = TRAIT_DEFLECTS_PROJECTILES,
-		"TRAIT_TABLE_LEAP" = TRAIT_TABLE_LEAP,
```

Comment:
im not sure you meant to delete this

Reply:
No, `TRAIT_TABLE_LEAP`is now a mind trait, this one was just never removed